notification/cmd/main.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"os/signal"
+	"syscall"
+
+	"github.com/GrabItYourself/giys-backend/lib/logger"
+	"github.com/GrabItYourself/giys-backend/lib/rabbitmq"
+	"github.com/GrabItYourself/giys-backend/lib/rabbitmq/types"
+	"github.com/GrabItYourself/giys-backend/notification/internal/config"
+	"github.com/GrabItYourself/giys-backend/notification/internal/handler"
+	"github.com/pkg/errors"
+	amqp "github.com/rabbitmq/amqp091-go"
+)
+
+func main() {
+	// Context
+	ctx, cancel := signal.NotifyContext(context.Background(), syscall.SIGTERM, syscall.SIGINT)
+	defer cancel()
+
+	// Config
+	conf := config.InitConfig()
+
+	// Logger
+	logger.InitLogger(&conf.Log)
+
+	// Connect to RabbitMQ
+	conn, err := amqp.Dial(conf.RabbitMQ.URL)
+	if err != nil {
+		logger.Fatal(errors.Wrap(err, "Failed to connect to RabbitMQ").Error())
+	}
+	defer conn.Close()
+
+	emailConsumer, err := rabbitmq.NewConsumer(conn, "email", "email-consumer")
```

Comment:
the consumer name should indicate what service they are from, imo. Maybe use something like `notification-email-consumer`

Reply:
It's also common practice to set consumer name in config variables i think, but it's not a big deal since we only have 1 consumer in the entire system here.